autofocus/predict/app/requests/Validator.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC, abstractmethod
+
+from flask import abort, jsonify, make_response
+from flask_api import status
+
+
+ALLOWED_IMAGE_FILES = set(["png", "jpg", "jpeg", "gif", "bmp"])
```

Comment:
A PR that was merged after this one was submitted uses `mimetypes` for this purpose: https://github.com/uptake/autofocus/blob/master/autofocus/predict/app/app.py. Please follow that approach.

Reply:
Okay, I resolved that issue.